superset/assets/package.json

```diff
@@ -8,9 +8,9 @@
     "test": "spec"
   },
   "scripts": {
-    "tdd": "mocha --require ignore-styles --compilers js:babel-core/register --require spec/helpers/shim.js 'spec/**/*_spec.*' --watch --recursive",
-    "test": "mocha --require ignore-styles --compilers js:babel-core/register --require spec/helpers/shim.js 'spec/**/*_spec.*'",
-    "test:one": "mocha --require ignore-styles --compilers js:babel-core/register --require spec/helpers/shim.js",
+    "tdd": "jest --watch",
+    "test": "jest",
+    "test:one": "jest",
```

Comment:
should we remove this one since it's the same as `test`? I added `test:one` for mocha just a couple months ago.

Reply:
Yep, we can.